static/scripts/homework.js

```diff
@@ -458,7 +458,7 @@ $(document).ready(() => {
 				url: $buttonContext.attr('href'),
 				type: 'DELETE',
 				data: {
-					key: $buttonContext.data('file-key'),
```

Comment:
we should also remove the key from above 
https://github.com/hpi-schul-cloud/schulcloud-client/pull/2718/files#diff-57ed59bb7b2fc85a8b3f9fda6b1b935173f9d696bd651ebf04d93d8bcf787c15R423